docs/versioned-plugins/inputs/beats-v4.0.1.asciidoc

```diff
@@ -46,57 +46,57 @@ output {
 
 NOTE: The Beats shipper automatically sets the `type` field on the event.
 You cannot override this setting in the Logstash config. If you specify
-a setting for the <<plugins-inputs-beats-type,`type`>> config option in
+a setting for the <<{version}-plugins-inputs-beats-type,`type`>> config option in
 Logstash, it is ignored.
 
 IMPORTANT: If you are shipping events that span multiple lines, you need to
 use the https://www.elastic.co/guide/en/beats/filebeat/current/multiline-examples.html[configuration options available in Filebeat] to handle multiline events
 before sending the event data to Logstash. You cannot use the
-<<plugins-codecs-multiline>> codec to handle multiline events. Doing so will
+{logstash-ref}/plugins-codecs-multiline.html[Multiline codec plugin] codec to handle multiline events. Doing so will
```

Comment:
This is an external link, so we need to convert it to use the format for external links. We'll need to fix this in the asciidoc in the plugin repo and then use search/replace to fix it in older docs for the plugin.